lib/ocpp/v201/connectivity_manager.cpp

```diff
@@ -135,7 +135,7 @@ void ConnectivityManager::connect(std::optional<int32_t> configuration_slot_opt)
         // After the websocket gets closed a reconnect will be triggered
         this->websocket->disconnect(WebsocketCloseReason::ServiceRestart);
     } else {
-        this->try_connect_websocket();
+        this->websocket_timer.timeout([this] { this->try_connect_websocket(); }, std::chrono::seconds(0));
```

Comment:
@marcemmers I used `std::chrono::seconds(0)` instead of `WEBSOCKET_INIT_DELAY` because the `WEBSOCKET_INIT_DELAY` was only introduced to solve an issue with OCTT tests reconnects as far as I remember.

Reply:
Yes I don't think there would be a point delaying there. Although I think we should probably revisit misusing a timer for this at all, that would be out of scope for this.